pilot/pkg/model/authentication.go

```diff
@@ -77,7 +80,11 @@ func initAuthenticationPolicies(env *Environment) *AuthenticationPolicies {
 		policy.addRequestAuthentication(configs)
 	}
 
-	// TODO(diemtvu): populate mTLS mode from mesh config and namespace labels.
+	if configs, err := env.List(collections.IstioSecurityV1Beta1Peerauthentications.Resource().GroupVersionKind(), NamespaceAll); err == nil {
```

Comment:
i'll suggest to break the line length. also we should log if error occurs

Reply:
Done. Also return error back to caller.